### ISSUE TITLE
[Accessibility][CreateDashboardCenturion] Decorative images are not hidden from screen reader users

### ISSUE THREAD

**Issue by mnischay** [awslabs/iot-application#1839]
####  Notes and Advice 
The error triangle decorative image is not hidden from screen reader users. Images that are decorative and do not convey any meaningful information to users must be hidden from assistive technologies.
 Recommendation 
Use an empty string as text alternative aria-hidden="true". This will ensure that the image is not announced by assistive technologies.
 
####  Recommended (simplified) markup
 
<div role="img" aria-hidden="true">
 
In HTML, empty attributes have an implicit null value, so alt has the same effect as alt="".

#### Resources 
[Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)

### AC

- [ ] The above violation is mitigated
